runestone/common/project_template/conf.tmpl

```diff
@@ -107,6 +107,16 @@ rst_prolog = (
 """
 )
 
+# Select whether to use server-side grading where possible. Server-side grading
+# requires **all** the following:
+#
+# - The use of Runestone services (``eBookConfig.useRunestoneServices === true``)
+# - A valid login (``eBookConfig.loginRequired === true``)
+# - Logging enabled (``eBookConfig.logLevel  > 0``)
+#
+# The first two conditions cause the ``RunestoneBase.logBookEvent`` in ``runestonebase.js`` to post a student response to the server. The last conditions ensures that ``hsblog`` in ``ajax.py`` on the server will return a response containing grading information.
+runestone_server_side_grading = %(server_side_grading)s
+
```

Comment:
OK, so this is the explicit parameter to enable server side grading?

Reply:
Yes, for the Sphinx generation code/code in this repo. The server-side code (ajax.py) also needs to know that server-side grading is enabled, but (obviously) that needs to be discussed/addressed at https://github.com/RunestoneInteractive/RunestoneServer/pull/1128.